tests/integration/targets/mso_remote_location/tasks/main.yml

```diff
@@ -0,0 +1,267 @@
+# Test code for the MSO modules
+# Copyright: (c) 2022, Akini Ross (@akinross) <akinross@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: Test that we have an ACI MultiSite host, username and password
+  fail:
+    msg: 'Please define the following variables: mso_hostname, mso_username and mso_password.'
+  when: mso_hostname is not defined or mso_username is not defined or mso_password is not defined
+
+# CLEAN ENVIRONMENT
+
+- name: Set vars
+  set_fact: 
+    mso_info: &mso_info
+      host: '{{ mso_hostname }}'
+      username: '{{ mso_username }}'
+      password: '{{ mso_password }}'
+      validate_certs: '{{ mso_validate_certs | default(false) }}'
+      use_ssl: '{{ mso_use_ssl | default(true) }}'
+      use_proxy: '{{ mso_use_proxy | default(true) }}'
+      output_level: '{{ mso_output_level | default("info") }}'
+
+- name: Query all backups
+  cisco.mso.mso_backup:
+    <<: *mso_info
+    state: query
+  delegate_to: localhost
+  register: backups
+
+- name: Ensure all backups with link to remote location ansible_test are removed
+  cisco.mso.mso_backup:
+    <<: *mso_info
+    backup: "{{ item.name }}"
+    state: absent
+  when:
+    - item.remoteLocationName is defined
+    - item.remoteLocationName == "ansible_test"
+  loop: "{{ backups.current }}"
+  delegate_to: localhost
+
+- name: Ensure remote location ansible_test is removed
+  cisco.mso.mso_remote_location:
+    <<: *mso_info
+    remote_location: ansible_test
+    state: absent
+  delegate_to: localhost
+
+- name: Configure remote location scp (check mode)
+  cisco.mso.mso_remote_location: &remote_location
+    <<: *mso_info
+    remote_location: ansible_test
+    remote_protocol: scp
+    remote_host: 173.36.219.25
+    remote_path: '/tmp'
```

Comment:
define a variable but use /tmp as default if variable is not defined by using:
```
'{{ mso_remote_location_path | default("/tmp") }}'

Reply:
ok